packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -145,6 +145,8 @@ export class TokenRatesController extends PollingControllerV1<
 
   #tokenPricesService: AbstractTokenPricesService;
 
+  #inProcessExchangeRateUpdate: Record<`${Hex}:${string}`, Promise<void>> = {};
```

Comment:
Nit: Plural?

``` suggestion
  #inProcessExchangeRateUpdates: Record<`${Hex}:${string}`, Promise<void>> = {};
```

Reply:
Good point, updated in https://github.com/MetaMask/core/pull/3635/commits/4b3bcde978833a139511c5402442c8004d0ac268